crates/gpui/Cargo.toml

```diff
@@ -26,6 +26,9 @@ anyhow.workspace = true
 async-task = "4.7"
 backtrace = { version = "0.3", optional = true }
 bitflags = "2.4.0"
+blade-graphics = { git = "https://github.com/kvark/blade", rev = "62eb18d312f720a5aac8f508fe223146a24fc7f0" }
+blade-macros = { git = "https://github.com/kvark/blade", rev = "62eb18d312f720a5aac8f508fe223146a24fc7f0" }
+bytemuck = "1"
```

Comment:
These added dependencies should be guarded for non-MacOS targets

Reply:
My plan is to use this path on all platforms. But I agree - for the sake of landing it sooner let's focus on Linux only.